### PR TITLE
Pangolin 3766 localized multiline text field

### PR DESCRIPTION
**Pull request by Sarah4VT**
#### Summary

Update the LocalizedMultilineTextField component to support an isCondensed and cacheMeasurements property that is passed down to the LocalizedMultilineTextInput to display the condensed layout.

## Description

Figma Design
image

Results
image

### CODE REVIEW

**Comment by Sarah4VT** [comment]
Do not merge this until the [dependent pr](https://github.com/commercetools/ui-kit/pull/2793) is merged.

**Comment by Sarah4VT** [comment]
This PR has the same height issue as the Input PR. Although oddly here I can't refresh storybook with isCondensed checked and have it render correctly on the first time for any fields that are on the page (it will render correctly the additional field after showing additional languages).

**Review by FilPob — APPROVED** [review]

File: .changeset/fuzzy-items-bow.md
Comment (CarlosCortizasCT) [inline .changeset/fuzzy-items-bow.md:1]: It seems this changeset does not apply to this PR, right?

Comment (Sarah4VT) [reply .changeset/fuzzy-items-bow.md:1]: Yes, i based this branch on the input branch and thought once that was merged, these would not be recognized as changes. That didn't happen! I am working on cleaning it up, but my morning meetings are going to take priority for a bit. Stay posted.

Comment (Sarah4VT) [reply .changeset/fuzzy-items-bow.md]: I should have fixed this now.

**Pull request merged**
